datus/storage/semantic_model/semantic_model_init.py

```
@@ -0,0 +1,151 @@
+        args: Command line arguments
+        agent_config: Agent configuration
+    """
+    df = pd.read_csv(args.success_story)
+
+    # Collect all SQL queries and questions
+    all_sqls = df["sql"].tolist()
+    all_questions = df["question"].tolist()
+
+    # Build comprehensive context from all rows
+    context_message = "Generate semantic models for the following SQL queries:\n\n"
+    for idx, (sql, question) in enumerate(zip(all_sqls, all_questions), 1):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)